README.md

```diff
@@ -1,4 +1,10 @@
-<p align="center"> <img alt="Space Station 14" width="880" height="300" src="https://raw.githubusercontent.com/space-wizards/asset-dump/de329a7898bb716b9d5ba9a0cd07f38e61f1ed05/github-logo.svg" /></p>
+<div class="header" align="center"> <img alt="Space Station 14" width="880" height="300" src="https://raw.githubusercontent.com/space-wizards/asset-dump/de329a7898bb716b9d5ba9a0cd07f38e61f1ed05/github-logo.svg" </p>
+
+![GitHub License](https://img.shields.io/github/license/space-wizards/space-station-14?style=for-the-badge&logo=github)
+![GitHub commit activity](https://img.shields.io/github/commit-activity/y/space-wizards/space-station-14?style=for-the-badge&logo=git&logoColor=white)
+![GitHub Issues or Pull Requests](https://img.shields.io/github/issues/space-wizards/space-station-14?style=for-the-badge&logo=helpdesk&logoColor=white)
```

Comment:
You can see all this information on the same GitHub page you are reading this read me from, no need to duplicate it. Also, clicking them does not bring you to the correct place which is confusing.

Remove it please.